README.md

```diff
@@ -1,8 +1,7 @@
-<p align="center">
+<p align=center>
   <a href="https://customer.io">
-    <img src="https://user-images.githubusercontent.com/6409227/144680509-907ee093-d7ad-4a9c-b0a5-f640eeb060cd.png" height="60">
+    <img src="https://avatars.githubusercontent.com/u/1152079?s=200&v=4" height="60">
```

Comment:
Same concern. Looks too small to me for README.

Reply:
I ran this by the brand design team. This is the size we're using on all our other libraries. https://customerio.slack.com/archives/C015KA32G05/p1714748585893119?thread_ts=1713910995.615139&cid=C015KA32G05

Reply:
Thanks for checking with the brand design team. The logo isn’t exactly my style though, but I’m not the expert 😄 Since it has their approval, we are good. 🚢